### PR TITLE
AutoRoom Generation (for the dev branche)

### PR DESCRIPTION
**Pull request by c-o-m-m-a-n-d-e-r**
Hey,
this can be added to the dev channel.

User @domeOo and i figured out when get_room_map is working and when not. I make an readme Point for that!

Also implement Zone cleaning

@afharo & @nicoh88

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,7 +93,13 @@ This plugin use the new [miio](https://github.com/aholstenson/miio) version 0.15
 | `waterBox` | false | when set to true, HomeKit shows an additional slider to control the amount of water released by the robot (only selected models like S5-Max). Currently in a beta state. |
 | `cleanword` | cleaning | used for autonaming the Roomselectors |
 | `rooms` | false | Array of ID / Name for a single Room. If set you have another switch for cleaning only this room |
+| `autoroom` | false | when set to true, Rooms while generated from Robot. (only S6) |
```
Comment (afharo) [inline README.md]: while = will be? 😉

File: config.schema.json [hunk 2]
```diff
@@ -75,6 +75,25 @@
...
+        "items": {
+          "title": "Zones",
+          "type": "object",
+          "properties": {
+            "name": {
+              "name": "Name",
+              "type": "string"
+            },
+            "zone": {
+              "name": "Zoon",
```
Comment (afharo) [inline config.schema.json]: Zoon? Is that German? :)

File: README.md (hunk 1 above)
Comment (c-o-m-m-a-n-d-e-r) [reply README.md]: yes, sometimes my fingers are too fast ... you are right : will be

File: config.schema.json (hunk 2 above)
Comment (c-o-m-m-a-n-d-e-r) [reply config.schema.json]: Typo ... Zone ... 

I will update both...

**Comment by nicoh88** [comment]
🌮 MERGE!

If it the #master tested by two/three users, we can publish 0.7.3 @ npm.
Please info to me.

```
sudo su -
npm install -g nicoh88/homebridge-xiaomi-roborock-vacuum#master
```

**Comment by nicoh88** [comment]
Sorry, again for me:

Rooms only works with S5 / S50 (firmware higher 2004) und S6?
AutoRoom only works with S6?

**Comment by c-o-m-m-a-n-d-e-r** [comment]
yes rooms are only working with s5 latest firmwarre and s6

autoroom only works with s6 and named rooms (info for that is in this readme update :-) )

autoroom will work for s5 too if xiaomi added the naming feature to the firmware or change the api response ... normally not a big deal

**Pull request merged**